yktrace/src/tir.rs

```diff
@@ -619,11 +629,33 @@ impl TirOp {
 }
 
 #[cfg(test)]
-mod tests {
+pub mod test_helpers {}
+
+#[cfg(test)]
+pub mod tests {
     use super::TirTrace;
-    use crate::{sir::SIR, start_tracing, TracingKind};
+    use crate::{sir::SIR, start_tracing, trace_debug, TracingKind};
+    use fm::FMBuilder;
+    use regex::Regex;
     use test::black_box;
 
+    /// Fuzzy matches the textual TIR for the trace `tt` with the pattern `ptn`.
+    pub fn assert_tir(ptn: &str, tt: &TirTrace) {
+        let ptn_re = Regex::new(r"%.+?\b").unwrap(); // Names are words prefixed with `%`.
+        let text_re = Regex::new(r"\$?.+?\b").unwrap(); // Any word optionally prefixed with `$`.
+        let matcher = FMBuilder::new(ptn)
+            .unwrap()
+            .name_matcher(Some((ptn_re, text_re)))
+            .distinct_name_matching(true)
+            .build()
+            .unwrap();
+
+        let res = matcher.matches(&format!("{}", tt));
+        if let Err(e) = res {
+            panic!(format!("{}", e));
```

Comment:
This should be `panic!("{}", e)`. The panic! variant you are using will likely be deprecated in the future.